packages/target-nodejs-sdk/src/target.js

```diff
@@ -128,6 +132,13 @@ export function executeDelivery(options, decisioningEngine) {
         Messages.RESPONSE_RECEIVED,
         JSON.stringify(response, null, 2)
       );
+
+      if (deliveryMethod.decisioningMethod === DECISIONING_METHOD.SERVER_SIDE) {
```

Comment:
Do you think we could remove the if statement and handle ODD telemetry here too?  Since inside the NotificationProvider the sendNotifications() call will call executeDelivery() for the ODD use case.  Feasible? Thoughts?